src/main/cpp/Robot.cpp

```diff
@@ -122,7 +122,14 @@ void Robot::TeleopInit()
     if (config != nullptr)
     {
         auto noteMgr = config->GetMechanism(MechanismTypes::MECHANISM_TYPE::NOTE_MANAGER);
-        if (noteMgr != nullptr)
+        bool allSensorsOff = ((m_noteManagerGen->getfeederSensor()->Get() == false) &&
+                              (m_noteManagerGen->getlauncherSensor()->Get() == false) &&
```

Comment:
inconsistent variable names; m_noteManagerGet has not been initialized yet.  There is no guarantee that getxxxsensor will not return a nullptr, so it needs to be checked before calling the get method.  

Then  boolean compares don't need == false, just user ! before it

I don't think you need a classwide variable either